demos/imagenet_classification_demo/include/grid_mat.hpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2018 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <set>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+
+class GridMat {
+public:
+    explicit GridMat(const cv::Size maxDisp = cv::Size{1920, 1080}, unsigned size = 2): size{size}, currSourceID{0} {
+        cellSize = cv::Size{int(maxDisp.width / size), int(maxDisp.height / size)};
+        rectangleHeight = maxDisp.height / 25;
+
+        for (size_t i = 0; i < size; i++) {
+            for (size_t j = 0; j < size; j++) {
+                cv::Point p;
+                p.x = cellSize.width * i;
+                p.y = rectangleHeight + (cellSize.height * j);
+                points.push_back(p);
+            }
+        }
+
+        outImg.create((cellSize.height * size) + rectangleHeight, cellSize.width * size, CV_8UC3);
+        outImg.setTo(0);
+    }
+
+    unsigned getSize() {
+        return size;
+    }
+
+    cv::Size getCellSize() {
+        return cellSize;
+    }
+
+    void listUpdate(std::queue<cv::Mat>& frames) {
+        while (!frames.empty()) {
+            updateList.push_back(frames.front());
+            frames.pop();
+        }
+    }
+
+    void textUpdate(double overSPF){
+        // Draw a rectangle
+        size_t colunmNum = outImg.cols;
```

Comment:
You use this var only once. Maybe better just use `outImg.cols` instead.

Reply:
Done